<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>521</width>
      <height>271</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Result</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="push_back">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>push_back()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_front">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>push_front()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pop_back">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>pop_back()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pop_front">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>pop_front()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="empty">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>empty()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>clear()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="size">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>size()</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
